app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
giải thích code `||=` giúp chị nghe nào

Reply:
||= là một toán tử gán điều kiện . Nó có nghĩa là nếu vế trái nil hoặc undefined hoặc false , thì nó sẽ đánh giá vế phải và gán giá trị của vế phải vào đối tượng ở vế trái.
ví dụ : a || =b thì đầy đủ sẽ là a || a=b